test/parallel/test-repl-unexpected-token-recoverable.js

```diff
@@ -12,21 +12,21 @@ const child = spawn(process.execPath, args);
 
 const input = 'var foo = "bar\\\nbaz"';
 // Match '...' as well since it marks a multi-line statement
-const expectOut = /^> \.\.\. undefined\n/;
+const expectOut = '> ... undefined\n';
 
 child.stderr.setEncoding('utf8');
-child.stderr.on('data', function(c) {
+child.stderr.on('data', (d) => {
   throw new Error('child.stderr be silent');
 });
 
 child.stdout.setEncoding('utf8');
 let out = '';
-child.stdout.on('data', function(c) {
-  out += c;
+child.stdout.on('data', (d) => {
+  out += d;
 });
 
-child.stdout.on('end', function() {
-  assert(expectOut.test(out));
+child.stdout.on('end', () => {
+  assert(out.includes(expectOut));
```

Comment:
Nit: these changes are all independent. Only the regular expression would have to be changed a tiny bit (only the `^` would have to be removed). The same in `test/parallel/test-repl-harmony.js`.